src/lib/settings/settingsUtils.ts

```diff
@@ -311,6 +311,24 @@ export function getAllCurrentSettings(
   return currentSettings
 }
 
+export function clearSettingsAtLevel(
+  allSettings: typeof settings,
+  level: SettingsLevel
+) {
+  Object.entries(allSettings).forEach(([category, settingsCategory]) => {
+    const categoryKey = category as keyof typeof settings
+    Object.entries(settingsCategory).forEach(
+      ([_, settingValue]: [string, Setting]) => {
+        if (settingValue[level] !== undefined) {
+          settingValue[level] = undefined
+        }
```

Comment:
Why not just `settingValue[level] = undefined` ? The code basically says "if it's set, unset it". Computationally you're spending more cycles on 1. doing equality computation and 2. doing the conditional check, vs just 1. assignment. 

Simply setting to undefined will be faster and clearer I'd think.

Reply:
Oh duh, thank you